databroker/broker.py

```diff
@@ -418,53 +502,58 @@ def dynamic_alias(self, key, func):
         self.aliases[key] = func
 
     def __call__(self, text_search=None, **kwargs):
-        """Given search criteria, find Headers describing runs.
+        """Search runs based on metadata.
 
-        This function returns a list of dictionary-like objects encapsulating
-        the metadata for a run -- start time, instruments used, and so on.
-        In addition to the Parameters below, advanced users can specifiy
-        arbitrary queries that are passed through to mongodb.
+        This function returns an iterable of :class:`Header` objects. Each
+        Header encapsulates the metadata for a run -- start time, instruments
+        used, and so on, and provides methods for loading the data. In
+        addition to the Parameters below, advanced users can specifiy arbitrary
+        queries using the MongoDB query syntax.
 
         Parameters
         ----------
         text_search : str, optional
             search full text of RunStart documents
-        start_time : time-like, optional
-            Include Headers for runs started after this time. Valid
-            "time-like" representations are:
-                - float timestamps (seconds since 1970), such as time.time()
```

Comment:
something like this:  
https://github.com/NSLS-II/databroker/blob/master/databroker/headersource/core.py#L767